Sources/Paywall/Paywall/Paywall View Controller/SWPaywallViewController.swift

```diff
@@ -49,8 +49,7 @@ final class SWPaywallViewController: UIViewController {
 
   // Views
 	lazy var shimmerView = ShimmeringView(frame: self.view.bounds)
-  private lazy var eventHandler = WebEventHandler(delegate: self)
-  lazy var webView = SWWebView(delegate: eventHandler)
+  lazy var webView = SWWebView(delegate: self)
```

Comment:
I believe I needed to keep the eventHandler as a separate object due to some bug in iOS. Something relating to retain cycles / webview being deallocated (not sure, will follow up)

What was the reason for this refactor? Just general clean up?